autodispose-android/src/main/java/com/uber/autodispose/android/ViewScopeProvider.java

```diff
@@ -33,7 +34,8 @@
  *      .empty();
  * </code></pre>
  */
-public class ViewScopeProvider implements LifecycleScopeProvider<ViewLifecycleEvent> {
+@UiThread
```

Comment:
odd to see this on a class. will this just apply it to all methods in the class?